### ISSUE TITLE
Greeh phi symbol U+03D5 missing from TTF format

### ISSUE THREAD

**Issue by mdealencar** [be5invis/Iosevka#689]
From the  issue  #565, it seems like that there is a glyph for U+03D5 in Iosevka. But I'm using the ttf fonts and this symbol is not being found (some programs replace it with another fonts' glyph, others with a placeholder).

---

* Your font version: 3.6.2
* Your font variant:
* Your operating system (name and version): Debian Linux testing
* Your application using Iosevka: terminal, jupyter-lab, xelatex

**Comment by be5invis** [comment]
Tested the latest build and the glyph is there. Could you please perform a clean uninstall, clear font cache, and reinstall?

**Comment by zero00072** [comment]
There is nothing wrong at mine, too.
![man](https://user-images.githubusercontent.com/2072036/94517163-ead36700-0259-11eb-896e-e74f8a2863c7.png)
![result](https://user-images.githubusercontent.com/2072036/94516545-a2677980-0258-11eb-9f2f-601407e7db86.png)

**Comment by mdealencar** [comment]
Thanks for the reply. I found the problem with another font that also has the glyph. It must be some problem in my system.

The other Greek letters work fine (such as the phi's at U+03C6 φ and U+03A6 Φ). As a workaround, I' using the phi at U+0278 ɸ.

I don't understand enough about the handling of unicode on Linux to figure this out.

**Comment by alerque** [comment]
You probably have a special confutation stuffed in fontconfig to override the font for a specific character or range. Sometimes misguided package maintainers think it's cute to package things like this with fonts installation. Some *Magic Greek Font™* that you installed packages for may have thought it in your best interest to hijack this symbol. What Linux distro are you using?

**Comment by be5invis** [comment]
Closing as external.

**Comment by mdealencar** [comment]
I apologize for the bogus bug report.

I got it sorted out now: when I was looking for the phi symbol ϕ (instead of the φ U+03C6 I was getting with the \<dead_greek\>+\<f\> combination), I used "\phi" with MathJax to get the desired phi shape (𝜙) and copy-pasted it. Later, when I started having problems, I found out about U+03D5 (ϕ) and I wrongly assumed I was using it, when 𝜙 is actually code U+1D719 (_Mathematical Italic Phi Symbol_) in the _Mathematical Alphanumeric Symbols_ block.

I wish I knew a better way of converting from the character to its code, but I will leave here what finally got me on the right track:
In ipython (or jupyter):
```python
In [1]: hex(ord('𝜙'))
Out[1]: '0x1d719'
```
